packages/keycodes/src/index.js

```diff
@@ -216,3 +224,27 @@ export const isKeyboardEvent = mapValues( modifiers, ( getModifiers ) => {
 		return event.key === character;
 	};
 } );
+
+/**
+ * Determine if an event is a "delete" keyboard event.
+ * This includes keypresses to `delete`, `backspace`, `ctrl+d`, `ctrl+h`, etc.
+ * @param {SyntheticEvent} event A synthetic keyboard event.
+ * @return {boolean} True if the event is a "delete" keyboard event, otherwise false.
+ */
+export const isDeleteKeyboardEvent = ( { keyCode, ctrlKey } ) => (
+	keyCode === DELETE ||
+	keyCode === BACKSPACE ||
+	// On Apple devices, treat control + d & control + h as delete events
+	( isAppleOS && ctrlKey && [ D, H ].includes( keyCode ) )
```

Comment:
Again, I'm not sure if this should be generally applied regardless of OS.

We should either take out the `isAppleOS` check or apply it to `isReverseDeleteKeyboardEvent` as well.  Open to suggestions!

Reply:
I went ahead and removed it for now so this is easier to test on other environments.